CVLByExamples/ReadOnlyReentrancy/certora/conf/BuiltinROReentrancyVulnerableBankFixed.conf

```diff
@@ -0,0 +1,16 @@
+{
+    "files": [
+        "contracts/VulnerableBankFixed.sol"
+    ],
+    "loop_iter": "3",
+    "optimistic_loop": true,
+    "process": "emv",
+    "solc": "solc8.17",
+    "disable_auto_cache_key_gen": true,
```

Comment:
remove

Reply:
But view reentrancy does not work with cache yet.

Reply:
why should we explicitly disable it though?

Reply:
Because it doesn't find the bugs with the cache. As if the cashe uses the code without instrumentation.